authorization-server/src/modules/auth/strategies/auth.strategy.ts

```diff
@@ -105,7 +105,9 @@ export class AuthStrategy extends PassportStrategy(LoginStrategy, 'login') {
         : '';
 
     return {
-      url: `https://${configService.get<string>('IPFS_HOST')}${port}`,
+      url:
+        `${configService.get<string>('IPFS_PROTOCOL')}://` +
+        `${configService.get<string>('IPFS_HOST')}${port}`,
```

Comment:
no, actually `port` contains this. And it could be empty if the default 443 port has been used when `IPFS_HOST` was empty. But, actually, now not only https can be used as a protocol. I need to rewrite this logic here and change `IPFS_HOST` to be required. 

Thanks for noticing this!

Reply:
done